arm/Microsoft.EventHub/namespaces/networkRuleSets/deploy.bicep

```diff
@@ -0,0 +1,69 @@
+@description('Conditional. The name of the parent event hub namespace. Required if the template is used in a standalone deployment.')
+param namespaceName string
+
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+@description('Optional. This determines if traffic is allowed over public network. Default it is "Enabled". If set to "Disabled", traffic to this namespace will be restricted over Private Endpoints only.')
+param publicNetworkAccess string = 'Enabled'
+
+@allowed([
+  'Allow'
+  'Deny'
+])
+@description('Optional. Default Action for Network Rule Set. Default is "Allow". Will be set to "Deny" if ipRules virtualNetworkRules or are being used. This would render the namespace resources inaccessible for data-plane requests if ipRules or virtualNetworkRules are not used and PublicNetworkAccess is set to "Disabled"')
+param defaultAction string = 'Allow'
+
+@description('Optional. List of IpRules. When used, defaultAction will be set to "Deny" and publicNetworkAccess will be set to "Enabled".')
```

Comment:
```suggestion
@description('Optional. List of IpRules. When used, the defaultAction will be set to "Deny" and publicNetworkAccess will be set to "Enabled" unless configured otherwise..')
```